src/main.ts

```diff
@@ -22,19 +23,18 @@ async function run(): Promise<void> {
 
     const issueKey = githubConnector.getIssueKeyFromTitle();
 
-    if (!issueKey) {
-      console.log(`JIRA key was not found`);
-      process.exit(0);
-    }
+    if (!issueKey) throw core.error;
 
     console.log(`JIRA key -> ${issueKey}`);
 
-    const details = await jiraConnector.getTicketDetails(issueKey);
-    await githubConnector.updatePrDetails(details);
+    await jiraConnector.getTicketDetails(issueKey);
   } catch (error) {
-    console.log({ error });
-    core.setFailed(error.message);
-    process.exit(1);
+    console.log('JIRA key was not found');
+    if (FAIL_WHEN_JIRA_ISSUE_NOT_FOUND) {
+      console.log({ error });
+      core.setFailed(error.message);
```

Comment:
would error has a message if it fails here?
https://github.com/cakeinpanic/jira-description-action/pull/12/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR26

Reply:
Yup, we get a message as `Error: Request failed with status code 404`

Reply:
Sample output - 
```Run keshavprasadms/jira-description-action@0.0.3
looking in:   fix: some bug fixes new cool  
looking in: SB-16033
JIRA key -> SB-16033
{
  errorMessages: [ 'Issue does not exist or you do not have permission to see it.' ],
  errors: {}
}
JIRA key was not found
{
  error: Error: Request failed with status code 404
      at createError (/home/runner/work/_actions/keshavprasadms/jira-description-action/0.0.3/lib/index.js:1:4651)
      at settle (/home/runner/work/_actions/keshavprasadms/jira-description-action/0.0.3/lib/index.js:1:121035)
      at IncomingMessage.handleStreamEnd (/home/runner/work/_actions/keshavprasadms/jira-description-action/0.0.3/lib/index.js:1:128998)
      at IncomingMessage.emit (events.js:215:7)
      at endReadableNT (_stream_readable.js:1184:12)
      at processTicksAndRejections (internal/process/task_queues.js:80:21) {
    config: {
      url: '/issue/SB-16033?fields=project,summary,issuetype',
      method: 'get',
      headers: [Object],
      baseURL: '***rest/api/3',
      transformRequest: [Array],
      transformResponse: [Array],
      timeout: 2000,
      adapter: [Function: httpAdapter],
      xsrfCookieName: 'XSRF-TOKEN',
      xsrfHeaderName: 'X-XSRF-TOKEN',
      maxContentLength: -1,
      validateStatus: [Function: validateStatus],
      data: undefined
    },
    request: ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 6,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: ***,
      _last: ***,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      useChunkedEncodingByDefault: false,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: 0,
      _hasBody: ***,
      _trailer: '',
      finished: ***,
      _headerSent: ***,
      socket: [TLSSocket],
      connection: [TLSSocket],
      _header: 'GET /rest/api/3/issue/SB-16033?fields=project,summary,issuetype HTTP/1.1\r\n' +
        'Accept: application/json, text/plain, */*\r\n' +
        'Authorization: Basic ***\r\n' +
        'User-Agent: axios/0.19.2\r\n' +
        'Host: org-ground.atlassian.net\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _onPendingData: [Function: noopPendingOutput],
      agent: [Agent],
      socketPath: undefined,
      method: 'GET',
      path: '/rest/api/3/issue/SB-16033?fields=project,summary,issuetype',
      _ended: ***,
      res: [IncomingMessage],
      aborted: false,
      timeoutCb: [Function: emitRequestTimeout],
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      _redirectable: [Writable],
      [Symbol(kNeedDrain)]: false,
      [Symbol(isCorked)]: false,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    response: {
      status: 404,
      statusText: 'Not Found',
      headers: [Object],
      config: [Object],
      request: [ClientRequest],
      data: [Object]
    },
    isAxiosError: ***,
    toJSON: [Function]
  }
}
Error: Request failed with status code 404
```